presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoInformer.java

```diff
@@ -52,21 +51,17 @@ public int getPrestoPort()
 
     /**
      * Returns whether Presto is running as a coordinator
-     *
+     *<p>
+     * Always check whether Presto is running using {@link PackageController#isRunning()}
```

Comment:
Because of two reasons:
1) `isRunning()` throws `PrestoManagerException`
I don't want `PrestoInformer` methods to throw `PrestoManagerException`
2) I don't want `isRunningCoordinator()` to return false if Presto itself is not running. I think it would be misleading.